ddtrace/contrib/django/patch.py

```diff
@@ -58,14 +58,17 @@
 )
 
 
+config_django = config.django
```

Comment:
what impact does this change alone have?

is it really that big of a cost for an attr lookup?

Reply:
This eats out about 0.1% of the overhead